src/box/wal.c

```diff
@@ -1194,7 +1194,12 @@ wal_write_to_disk(struct cmsg *msg)
 		err_code = JOURNAL_ENTRY_ERR_IO;
 		goto done;
 	}
-
+#if !NDEBUG
+	struct errinj *inj;
+	inj = errinj(ERRINJ_WAL_WRITE_DURATION, ERRINJ_DOUBLE);
+	if (inj->dparam > 0)
```

Comment:
I suppose it's better to introduce `ERROR_INJECT_SLEEP_FOR` macros in `errinj.h` in order to make it more reusable. Or we can modify the existing `ERROR_INJECT_SLEEP`